addon/formatters/-base.js

```diff
@@ -65,7 +65,7 @@ const FormatBase = Ember.Object.extend({
 });
 
 FormatBase.reopenClass({
```

Comment:
Actually, let me think about this some more.  There are trade offs to both approaches.